markdown_writer.py

```diff
@@ -12,8 +12,7 @@
         average_time_to_close: timedelta,
         average_time_to_answer: timedelta,
         num_issues_opened: int,
-        num_issues_closed: int,
-        file: file object = None
```

Comment:
@zkoppert I found these traces in the documentation.

Apparently at some point in the past, a file object was passed to the `write_to_markdown()`.
Do you happen to know anything about this?

As I am looking to add the feature to set a custom OUTPUT_FILE, I don't want to go down an implementation path that might have been already tried in the past (and did not work). :)

Reply:
Add a link to the commit where I removed that unused bit of documentation. GitHub did not give me a straightforward way to get there from my comment above.
https://github.com/github/issue-metrics/pull/160/commits/5c5cd84b95caffd33fc84b832bed9a79b5a16949

Reply:
Yeah I think that was aspirational. I don't think we actually wrote the code to support that.